src/Data/Yaml/Internal.hs

```diff
@@ -22,7 +22,8 @@ import Control.Applicative ((<|>))
 import Control.Exception
 import Control.Monad (when, unless)
 import Control.Monad.Trans.Resource (ResourceT, runResourceT)
-import Control.Monad.RWS
+import Control.Monad.State
```

Comment:
Can we use the strict state instance?

Reply:
I guess yes.  There is also other stuff that can be made strict.  The thing is that I didn't observe any meaningful difference by changing strictness.

I don't have a strong opinion.

If we decide to change strictness, would it make sense to do it as a separate change?

Reply:
I'd say do it right now. My default state transformer is _always_ the strict one. I only use lazy versions if I have a specific need (which I almost never do).